pyproject.toml

```diff
@@ -38,7 +37,7 @@ keywords = [
     "reinforcement learning",
     "synthetic data",
 ]
-license = { file = "LICENSE" }
+license = { text = "Apache-2.0 License" }
```

Comment:
I'd leave the reference to the actual file we have, as it was before

Reply:
As discussed, I'll leave the updated form, because this looks cleaner on PyPi.